components/butterfly/src/error.rs

```diff
@@ -43,29 +45,31 @@ pub enum Error {
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         let msg = match *self {
-            Error::BadMessage(ref err) => format!("Bad Message: {:?}", err),
-            Error::CannotBind(ref err) => format!("Cannot bind to port: {:?}", err),
+            Error::BadMessage(ref err) => format!("Bad Message, {:?}", err),
```

Comment:
Ah yeah, I should have made a comment about this - totally forgot about the change.

All of our servers log with the format of "msg, {errKey}={errMsg}". I do that on the servers so we can later easily setup patterns to match on in a log aggregator for things we care about being alerted on or just for ease of debugging.

We don't necessarily need to standardize on it here since these messages aren't going into a log aggregator. I just saw these and thought "oh that's a server and that's how the logs are supposed to be formatted".

Additionally, we can standardize on commas or colons or any other separator in the servers if you have a preference!

I'll change these back